ci-operator/jobs/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main-presubmits.yaml

```diff
@@ -49,7 +111,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
weird, you know why those were also updated?
should be as before.

Reply:
@viroel hey i just resolved the merge conflict and ran make update after that i am seeing these changes.